server/datastore/mysql/scripts.go

```diff
@@ -96,12 +96,31 @@ func (ds *Datastore) ListPendingHostScriptExecutions(ctx context.Context, hostID
     host_script_results
   WHERE
     host_id = ? AND
-    exit_code IS NULL AND
-    created_at >= DATE_SUB(NOW(), INTERVAL ? SECOND)`
+    exit_code IS NULL
+  ORDER BY
+    created_at ASC`
 
 	var results []*fleet.HostScriptResult
-	seconds := int(ignoreOlder.Seconds())
-	if err := sqlx.SelectContext(ctx, ds.reader(ctx), &results, listStmt, hostID, seconds); err != nil {
+	if err := sqlx.SelectContext(ctx, ds.reader(ctx), &results, listStmt, hostID); err != nil {
+		return nil, ctxerr.Wrap(ctx, err, "list pending host script results")
+	}
+	return results, nil
+}
+
+func (ds *Datastore) GetPendingHostScripts(ctx context.Context, hostID uint, scriptID uint) ([]*uint, error) {
```

Comment:
maybe `PendingScriptExecutionsForHost` or `IsExecutionPendingForHost`? this name confused me initially

Reply:
Yeah I was trying to come up with a better name and being so out of it on Friday I couldn't 🤦 
I like `IsExecutionPendingForHost`! I'll use that one